docker/wrapper.sh

```diff
@@ -281,18 +281,6 @@ then
     ln -s ../"${REV_READ}"
 
     oa index --no-pipe chloro "${FW_READ}" "${REV_READ}" 2>&1 | tee index.log
```

Comment:
Theoretically, we can remove the `index.log` file

Reply:
And what about the `--no-pipe` option?

Reply:
good points